contracts/FixedRateRewardsSource.sol

```diff
@@ -139,6 +135,11 @@ contract FixedRateRewardsSource is Governable, Initializable {
         // Update storage
         RewardConfig storage _config = rewardConfig;
         emit RewardsPerSecondChanged(_rewardsPerSecond, _config.rewardsPerSecond);
+        if (_config.rewardsPerSecond == 0) {
+            // When changing rate from zero to non-zero,
+            // Update lastCollect timestamp as well
```

Comment:
I only have 1 more nitpick: 
I would expand this comment to
```
/* This contract code allows for contract deployment & initialization and then the contract can be live for quite
 * some time before it is funded and `_rewardsPerSecond` are set to non 0 value. In that case the vesting period 
 * from contract initialization until now would be taken into account instead of the time since the contract has been
 * "activated" by setting the `setRewardsPerSecond`. To mitigate the issue we update the `_config.lastCollect`
 * to current time. 
 */
```

Reply:
This is done